sql/core/src/main/scala/org/apache/spark/sql/execution/SQLExecution.scala

```diff
@@ -46,7 +45,11 @@ private[sql] object SQLExecution {
       val executionId = SQLExecution.nextExecutionId
       sc.setLocalProperty(EXECUTION_ID_KEY, executionId.toString)
       val r = try {
-        val callSite = Utils.getCallSite()
+        // We first try to pick up any call site that was set previously, then fall back to
+        // Utils.getCallSite(); because call Utils.getCallSite() on continuous queries directly
+        // would give us call site like "run at <unknown>:0"
+        val callSite = sparkSession.sparkContext.getCallSite()
```

Comment:
@zsxwing sorry for the vague comments. The fall back logic is contained in the `sparkSession.sparkContext.getCallSite()` itself. I've updated the comments accordingly.